src/Orleans.PlatformServices/Orleans.PlatformServices.csproj

```diff
@@ -9,26 +9,7 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <TargetFramework />
-    <TargetFrameworks>net462;netstandard1.5</TargetFrameworks>
```

Comment:
why was this removed? it kills the reason to have PlatformServices in the first place if we do not multi-target. Let's not remove it, and we can decide to remove the project later if it's not required

Reply:
The functionality it provides is not needed anymore with NetStandard 2.0, that's why it was removed and yes the PlatformAssemblyLoader itself can be removed now, maybe we'll have need for different platforms when we are extending to *nix for example.

Reply:
should we delete the entire project then? As it is, it has no reason to exist, as the ONLY reason it existed is so that it can multi-target.

Reply:
happy for us to remove it - we can add it back if it's needed